snmp/datadog_checks/snmp/data/default_profiles/_generic-ups.yaml

```diff
@@ -0,0 +1,168 @@
+metric_tags:
+  - OID: 1.3.6.1.2.1.33.1.1.1
+    symbol: upsIdentManufacturer
+    tag: ups_ident_manufacturer
+  - OID: 1.3.6.1.2.1.33.1.1.2
+    symbol: upsIdentModel
+    tag: ups_ident_model
+  - OID: 1.3.6.1.2.1.33.1.1.5
+    symbol: upsIdentName
+    tag: ups_ident_name
```

Comment:
🥜 Maybe we can keep `metric_tags` section at the bottom of the file to be more consistent with other profiles.